design/thin-edge-actors-design.md

```diff
@@ -446,6 +446,37 @@ Even if details are missing, this gives us a sketch of the pieces making an acto
   * This is a place to adapt sent messages to the type actually expected by the receiver.
   * They can also act as facades that build and send messages on regular method invocations.
 
+## Detailed Proposal
+
+Key design ideas:
+
+* All the events, requests and responses that affect the behaviour of an actor (including timeouts and cancellations)
```

Comment:
The responses would be going to another actor's mailbox, right?

Reply:
Yes correct. I will add a diagram to make things more visual.